src/test/java/github/weichware10/analyse/HeatmapTest.java

```diff
@@ -33,22 +33,22 @@ public void createHeatmapShouldWork() {
      */
     @Test
     public void compHeatmapsShouldWork() {
-        Heatmap hm1 = new Heatmap(new Data(ToolType.CODECHARTS, 4), new ConfigHeatmap());
-        Heatmap hm2 = new Heatmap(new Data(ToolType.EYETRACKING, 2), new ConfigHeatmap());
+        Heatmap hm1 = new Heatmap(new TrialData(ToolType.CODECHARTS, "4", "3"), new ConfigHeatmap());
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 101).